### ISSUE TITLE
Timesheets not working when projects are "closed"

### ISSUE THREAD

**Issue by BenoitGielly** [cgwire/kitsu#1123]
**Context**

Studio name: Stim Studio
Versions: Kitsu 0.17.7 / Zou 0.17.11
Production type: Animation / VFX

**Describe the bug**

When an artist has some timesheets added to "projectA", as soon as we close this project on Kitsu, the timesheet side page won't open and the following error happens in the Browser's console:

```
vue.js:5108 [Vue warn]: Error in render: "TypeError: Cannot read properties of undefined (reading 'production_type')"

found in

---> <TimespentTaskList>
       <PeopleTimesheetInfo>
         <People>
           <MainWrapper>
             <App>
               <Root>
gt @ vue.js:5108
vue.js:3732 TypeError: Cannot read properties of undefined (reading 'production_type')
    at xe.getTaskPath (TimeSpentTaskList.vue:140:1)
    at TimeSpentTaskList.vue:158:1
    at Proxy.la (vue.js:1861:24)
    at TimeSpentTaskList.vue:158:1
    at Proxy.la (vue.js:1861:24)
    at TimeSpentTaskList.vue:158:1
    at Proxy.la (vue.js:1861:24)
    at Proxy.F (TimeSpentTaskList.vue:158:1)
    at g._render (vue.js:2509:30)
    at xe.$ (vue.js:2948:29)
ss @ vue.js:3732
```

**Screenshots**
![kitsu_timesheets_error](https://github.com/cgwire/kitsu/assets/17833821/0193fe80-bf67-419c-b972-82576ea55e84)

**Desktop:**

 - OS: Windows 10 Pro
 - Browser Chrome
 - Version 114.0.5735.199

**Comment by NicoPennec** [comment]
Thanks @BenoitGielly for your detailed report 🙏 
The issue is fixed on [Kitsu >= 0.17.9](https://github.com/cgwire/kitsu/releases).

**Comment by BenoitGielly** [comment]
Super! j'ai mis à jour et ca fonctionne parfaitement, merci beaucoup!
